test/e2e/specs/editor/blocks/cover.spec.js

```diff
@@ -0,0 +1,241 @@
+/**
+ * External dependencies
+ */
+import path from 'path';
+import fs from 'fs';
+import os from 'os';
+import { v4 as uuid } from 'uuid';
+
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+async function getTestImage() {
```

Comment:
Could we follow what other [tests](https://github.com/WordPress/gutenberg/blob/trunk/test/e2e/specs/editor/blocks/image.spec.js) do and move them into a [POM](https://playwright.dev/docs/pom)-style class at the bottom of the file? It's just a coding style we prefer when writing e2e tests to keep them consistent 😆 .

Reply:
Thanks for suggestion, wasn't aware this is what other tests did. 

Addressed in 071c31b4fe4685041cbd67bfe90e7d5f27131a0c.